bitmovin/notifications/types.ts

```diff
@@ -1,6 +1,7 @@
 // email notifications
 export enum EventTypes {
-  LIVE_INPUT_STREAM_CHANGED = 'LIVE_INPUT_STREAM_CHANGED'
+  LIVE_INPUT_STREAM_CHANGED = 'LIVE_INPUT_STREAM_CHANGED',
+  ERROR = 'ERROR'
```

Comment:
should be `ENCODING_ERROR`

Reply:
This is done, not sure why github doesn't show this as outdated